### ISSUE TITLE
Is there a way to add this as a Android Studio project dependency?

### ISSUE THREAD

**Issue by ZOlbrys** [DreaminginCodeZH/betterpickers#1]
If not, how would I include this into my project? Download a local copy, add it as a module to my application?

Thanks!

**Comment by DreaminginCodeZH** [comment]
Yes, I did not publish this to maven central, so you can either add a local maven repo, or use [JitPack](https://jitpack.io/#DreaminginCodeZH/betterpickers/bdf320120e).

**Comment by ZOlbrys** [comment]
Thanks, the JitPack method is awesome!  Very cool.

One more question, when I use the ExpirationPicker, I am allowed to use dates before today, i.e. 01/2015 is allowed.  Wouldn't it make more sense to only allow dates from the current month? So for today the earliest ExpirationPicker value should be 08/2015.  Not sure if it's possible to implement this... not a huge issue I guess either, but either way, thanks!

**Comment by DreaminginCodeZH** [comment]
The upstream now has a minimum year setting (but not a minimum month) ([source](https://github.com/derekbrameyer/android-betterpickers/tree/master/library/src/main/java/com/codetroopers/betterpickers/expirationpicker)) after I forked it last year, so if you really need it, you can fork this repo and add your own logic according to the implementation from upstream, then use JitPack for your own fork.
